client/src/Components/forms/FormInputs.js

```diff
@@ -207,6 +204,18 @@ const FormInputs = (type, data, {...props}) =>{
             label: 'Type',
             value: formik.values.type,
             onChange: formik.handleChange('type'),
+            onBlur: () =>{ 
```

Comment:
This is the bug when changing the "type" i.e. dog, cat, other.  I tried to update the options list in the Breed section but formik wont let me run multiple functions in the onChange prop. 
normally I would do : `onChange={()=>{ func1(); func2();}}`
Instead I used the onBlur to update but the problem with onBlur is that it not updating the select dropdown until I click on the dropdown items or anywhere else on the screen. If you click anywhere other than the dropdown then there is no issue. If you do click the dropdown it will through an error because the value doesn't exist.